pipeline/packer/jenkins-master.json

```diff
@@ -0,0 +1,51 @@
+{
+  "_description": "This template produces a Jenkins server.",
+  "variables": {
+    "repository": "{{env `DOCKER_REPOSITORY`}}",
+    "tag": "{{env `BUILD_TAG`}}",
+    "docker_registry_url": "{{env `DOCKER_REGISTRY_URL`}}"
+  },
+  "builders": [
+    {
+      "type": "docker",
+      "image": "jenkins/jenkins:lts",
```

Comment:
TODO-possibly switch to using a specific version, rather than the latest lts release.